server/apps/distributed-app/sample-configuration/jmap.properties

```diff
@@ -6,6 +6,8 @@ enabled=true
 tls.keystoreURL=file://conf/keystore
 tls.secret=james72laBalle
 
+delay.sends.enabled=true
+
```

Comment:
Please note that this only work with Pulsar and not RabbitMQ.
BTW did we document this `delay.sends.enabled` configuration property in `jmap.adoc` and others?

Reply:
Yes, we have